### PR TITLE
Fix using through SwiftPM

### PR DESCRIPTION
**Pull request by Cykelero**
The CrashReportWindowController wouldn't find its nib when CrashReporter was loaded through SwiftPM.

- Bump package tools version (see: https://developer.apple.com/documentation/xcode/bundling-resources-with-a-swift-package)
- Disable module auto discovery in nib
- Change window controller initialization code to manually load the nib
	- Add CrashReporterBundle to expose correct current bundle

### CODE REVIEW

File: Sources/CrashReporter/UI/CrashReportWindowController.swift [hunk 1]
```diff
@@ -19,14 +19,21 @@ final class CrashReportWindowController: NSWindowController, NSWindowDelegate {
...
 
         self.crashLogText = crashLogText
         self.crashLogSender = crashLogSender
         self.privacyPolicyURL = privacyPolicyURL
         self.collectEmailSetting = collectEmailSetting
         self.sendReportsAutomaticallySetting = sendReportsAutomaticallySetting
+		
+		// Replicate default init behavior
+		window?.delegate = self
+		windowDidLoad()
```
Comment (DivineDominion) [inline Sources/CrashReporter/UI/CrashReportWindowController.swift:36]: I'm fine with inlining the function here, too, now that we are doing everything manually anyway. Less indirection, and all properties are set by this point anyway.

**Review by DivineDominion — APPROVED** [review]
Code looks good, thank you!

Could you replace tabs with 4 spaces real quick for consistency? I believe it's required to use `swift format`, now shipping with Xcode 16+, in the future, but that's not going to run on auto-save. Ugh. Sorry for the nit I'm picking here :(

**Comment by DivineDominion** [comment]
`⌃⇧I` apparently also works to quickly fix indentation and such: https://troz.net/post/2024/swift_format/

**Comment by Cykelero** [comment]
No problem—I usually try to match indentation style, but forgot.

I applied swift-format, which changed more than indentation. Do tell if that's not what you had in mind!

**Comment by DivineDominion** [comment]
Thanks! The noise of `swift-format`'ing doesn't matter, as long as the result looks ok :)

**Pull request merged**